content/spin/v2/python-components.md

```diff
@@ -114,55 +96,61 @@ component = "hello-world"
 [component.hello-world]
 source = "app.wasm"
 [component.hello-world.build]
-command = "spin py2wasm app -o app.wasm"
+command = "componentize-py -w spin-http componentize -p . -p ../../src app -o app.wasm"
 ```
 
+The `requirements.txt` by default contains the references to the `spin-sdk` and `componentie-py` packages. 
```

Comment:
Oh does that mean it will auto install and they don't need to do anything about it?  In that case never mind.

Reply:
They will have to run a `pip install -r requirements.txt`. I will make the change.